base/model/src/test/java/org/eclipse/ditto/base/model/common/LikeHelperTest.java

```diff
@@ -29,6 +30,15 @@ public void testWildcards() {
         assertExpression("foo..bar", "*bar", true);
         assertExpression("foo.bar.baz", "bar", false);
         assertExpression("foo.bar.baz", "*bar*", true);
+        //case insensitive test cases appending this  (?i) before text
```

Comment:
I would move the case insensitive tests to another test method.
Also, it seems the test is failing.

Reply:
Yes I right some tests are failing because of this (?i). I will use flag Pattern.CASE_INSENSITIVE with Pattern.compile